inference-engine/src/vpu/graph_transformer/src/middleend/pass_manager.cpp

```diff
@@ -202,13 +202,6 @@ PassSet::Ptr PassManager::buildMiddleEnd() {
         ADD_DUMP_PASS("replaceWithSCReLU");
     }
 
-    //
-    // Replace StridedSlice to other stages
-    //
-
-    ADD_PASS(stridedSlice);
-    ADD_DUMP_PASS("stridedSlice");
```

Comment:
I'd like to see performance validation

Reply:
It's in progress.

Reply:
Done. No issues connected with StridedSlice.